.travis.yml

```
@@ -7,7 +7,7 @@ python:
 install:
   - pip install --upgrade pip
   - pip install cython
-  - pip install -U -r requirements.txt pytest-mock .
+  - pip install -U -r requirements.txt pytest-mock snapshottest .
```

```suggestion
  - pip install -U -r requirements.txt .
```

and add `snapshottest` to the `requirements-dev.txt`

---

As discussed, let's leave it like it is. Also, `snapshottest` is already in `requirements-dev.txt`.